oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -216,6 +216,13 @@ spring-cloud-gateway:
 RESTEasy:
   id: 62
   languages: Java
+SolrJ:
+  id: 63
+  languages: Java
+Solr:
```

Comment:
Is this what you will have in the other PR #2853 ? If so, leave it to where it should be